tests/node-e2e/outgoing/test_outgoing_traffic_make_request_after_listen.mjs

```diff
@@ -19,22 +19,22 @@ const makeRequest = () => {
     });
 
     response.on("error", (fail) => {
-      console.error(`>> response failed with ${fail}`);
-
-      process.exit(-1);
+      process.stderr.write(`>> response failed with ${fail}`);
```

Comment:
When do you use process.stderr.write and when do you use console.log?

Reply:
Tbh this should all be the same (either write to stdout(err), or console). Changed to be all `console.log` and `console.error`.